schema_editor/app/scripts/resources/boundaries-service.js

```diff
@@ -0,0 +1,29 @@
+(function () {
+    'use strict';
+
+    /* ngInject */
+    function Boundaries($resource, Config) {
+        var urlString = Config.api.hostname + '/api/boundaries/:id';
+
+        var resource = {
+          'query': stdResource.query,
+          'get': stdResource.get,
+          'post': postWithFile,
+          'remove': stdResource.remove,
+          'delete': stdResource.delete
+        };
+
+        var stdResource = $resource(urlString, {id: '@uuid'}, {
+            query: {
+                method: 'GET',
+                transformResponse: function(data) { return angular.fromJson(data).results; },
+                isArray: true
+            }
+        });
+
+        return resource;
```

Comment:
Ah ok. That would work, although that directive is designed with async upload operations in mind and this is a synchronous one so to keep things simple you could probably just use the angular-file-upload plugin directly to start if necessary.